specification/components/schemas/CodeSystemConcept.yaml

```diff
@@ -32,29 +33,38 @@ properties:
       | Clinic Type                     | General Urology               |
       | Appointment Cancellation Reason | Inappropriate service         |
+      | Referral Cancellation Reason    | Intend to go private          |
       
     type: string
   property:
     description: |
 
       Some properties are only returned for certain code systems
       
-      | Property                                 | Code System                     |
-      | -----------------------------------------| ------------------------------- |
-      | EffectiveFrom                            | All Code Systems                |
-      | EffectiveTo                              | All Code Systems                |
-      | Specialty                                | CLINIC-TYPE                     |
-      | AppointmentBookingCancellationReasonType | APPOINTMENT-CANCELLATION-REASON |
-      | WillCancelWholeRequest                   | APPOINTMENT-CANCELLATION-REASON |
-      | CommentIsMandatory                       | APPOINTMENT-CANCELLATION-REASON |
+      | Property                                 | Code System                                                 |
+      | -----------------------------------------| ----------------------------------------------------------- |
+      | EffectiveFrom                            | All Code Systems                                            |
+      | EffectiveTo                              | All Code Systems                                            |
+      | Specialty                                | CLINIC-TYPE                                                 |
+      | AppointmentBookingCancellationReasonType | APPOINTMENT-CANCELLATION-REASON                             |
+      | WillCancelWholeRequest                   | APPOINTMENT-CANCELLATION-REASON                             |
+      | Usage                                    | APPOINTMENT-CANCELLATION-REASON                             |
+      | CommentIsMandatory                       | APPOINTMENT-CANCELLATION-REASON/REFERRAL-CANCELLATION-REASON |
+      | businessFunction                         | APPOINTMENT-CANCELLATION-REASON/REFERRAL-CANCELLATION-REASON |
+      | CanCancelAppointment                     | REFERRAL-CANCELLATION-REASON                                 |
+      | RequestCancellationReferrerUsage         | REFERRAL-CANCELLATION-REASON                                 |
       
     type: array
     nullable: true
     items:
       anyOf:
-        - $ref: 'codings/CommentIsMandatory.yaml'
         - $ref: 'codings/WillCancelWholeRequest.yaml'
         - $ref: 'codings/EffectiveFrom.yaml'
         - $ref: 'codings/EffectiveTo.yaml'
         - $ref: 'codings/Specialty.yaml'
         - $ref: 'codings/AppointmentBookingCancellationReasonType.yaml'
+        - $ref: 'codings/AppointmentCancellationUsage.yaml'
+        - $ref: 'codings/CommentIsMandatory.yaml'
+        - $ref: 'codings/BusinessFunction.yaml'
```

Comment:
If this supports multiple business functions does it also support multiples of the other refs?